### PR TITLE
Support StackExchange, LastFM, and Instagram.

### PR DESCRIPTION
**Pull request by chrisvogt**
Adds support for StackExchange, Instagram, and Last.fm

add-support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,30 +4,32 @@ Responsive theme for [JsonResume](https://jsonresume.org/) inspired by card layo
...
-### Supported Social Profiles 
-* github
-* linkedin
-* stackoverflow
-* twitter
-* soundcloud
-* pinterest
-* vimeo
+### Supported Social Profiles
+* angellist
```
Comment (mudassir0909) [inline README.md]: :+1: for the alphabetical order

File: assets/less/theme.less [hunk 2]
```diff
@@ -26,4 +26,4 @@
 @import "sections/interests";
 @import "sections/references";
 
-@import "print";
```
Comment (mudassir0909) [inline assets/less/theme.less]: Was this removed accidentally ?

Comment (chrisvogt) [inline assets/less/theme.less]: Good catch, that wasn't removed intentionally.

File: assets/less/variables.less [hunk 3]
```diff
@@ -47,4 +50,4 @@
 
 // Label styles
 @theme-label-bg: lighten(@theme-blue, 15%);
-@theme-label-border-color: darken(@theme-label-bg, 20%);
```
Comment (chrisvogt) [inline assets/less/variables.less]: I'm not too certain this looks familiar either. Odd, both files have newline issues at the end. Perhaps Grunt is responsible for these.

**Comment by chrisvogt** [comment]
I feel like I'm going crazy @mudassir0909! I just checked out my branch and those lines do not appear removed. Also, if I [view the file for that commit](https://github.com/chrisvogt/jsonresume-theme-elegant/blob/9eadbcb9f172d7f2134af7bf0b84474d5d0f2fd7/assets/less/theme.less) I do not see those lines missing either.

**Comment by chrisvogt** [comment]
Oh, I see! I think it might be because there is a newline missing from each file, and so GitHub isn't noticing the very last lines.

File: assets/less/variables.less (hunk 3 above)
Comment (mudassir0909) [inline assets/less/variables.less]: Grunt doesn't modify less files, it just compiles them to a different file

**Comment by mudassir0909** [comment]
@chrisvogt Strange! But this commit https://github.com/chrisvogt/jsonresume-theme-elegant/commit/9eadbcb9f172d7f2134af7bf0b84474d5d0f2fd7 says so and the file doesn't :confused:

**Comment by chrisvogt** [comment]
I think this was a GitHub bug. I just modified those two files, adding a comment to one and swapping two lines in the other, and now it no longer thinks those two lines were removed.

**Comment by mudassir0909** [comment]
@chrisvogt Sorry was bit busy last couple of days, cool that it got resolved! probably the first time I'm witnessing a bug on github ;)

File: assets/less/variables.less [hunk 4]
```diff
@@ -46,5 +49,5 @@
 @skill-master-color: @theme-green;
 
 // Label styles
+@theme-label-border-color: darken(@theme-label-bg, 20%);
```
Comment (mudassir0909) [inline assets/less/variables.less]: I think this line should come after
```less
@theme-label-bg: lighten(@theme-blue, 15%);
```
else we might get variable undefined error

**Comment by mudassir0909** [comment]
@chrisvogt one minor change, everything else is fine. Let me know once done, I'll merge it right away

**Comment by mudassir0909** [comment]
@chrisvogt This change is live now, thanks for the pr!

**Pull request merged**